sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2350,7 +2350,7 @@ object functions {
   def unbase64(e: Column): Column = withExpr { UnBase64(e.expr) }
 
   /**
-   * Right-padded with pad to a length of len.
+   * Right-pad the string column with pad to a length of len.
```

Comment:
While here, let's make the description more complete in both cases, and consistent with the SQL usage doc:

```
    _FUNC_(str, len, pad) - Returns `str`, right-padded with `pad` to a length of `len`.
      If `str` is longer than `len`, the return value is shortened to `len` characters.
```